### PR TITLE
Fix host defaults

### PR DESCRIPTION
**Pull request by hpx7**

### CODE REVIEW

File: templates/base/api/base.ts.hbs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import axios from "axios";
 import * as T from "./types";
 
-export const COORDINATOR_HOST = process.env.COORDINATOR_HOST!;
-export const MATCHMAKER_HOST = process.env.MATCHMAKER_HOST!;
+export const COORDINATOR_HOST = process.env.COORDINATOR_HOST ?? "coordinator.hathora.dev";
```
Comment (dsiddharth) [inline templates/base/api/base.ts.hbs:4]: can we handle APP_SECRET and DATA_DIR in the same way? for consistency sake

I know they're not runtime args right now, but would prefer making them runtime args

Comment (hpx7) [reply templates/base/api/base.ts.hbs:4]: yeah can do that as a separate change

Comment (dsiddharth) [reply templates/base/api/base.ts.hbs:4]: and can we get rid of STORE_REGION?

**Review by dsiddharth — CHANGES_REQUESTED** [review]

**Review by dsiddharth — APPROVED** [review]
ok approving because the comments can be done in a separate PR

**Pull request merged**